### ISSUE TITLE
Testing: fix Travis CI failed on PHP 8.1

### ISSUE THREAD

**Issue by hidehalo** [hidehalo/nanoid-php#18]
## Detailed description

Fix Travis CI failed on PHP 8.1

## Context

After #16  implemented and merge into branch master, Travis CI failed on PHP 8.1. 

The reason is Travis CI doesn't provide an available building environment for PHP 8.1 on Ubuntu Xenial.

## Possible implementation

Use Travis CI [build matrix](https://docs.travis-ci.com/user/build-matrix/) configuration.

## Your environment

Travis CI PHP 8.1 on Ubuntu Xenial.
